panda/src/audio/audioSound.cxx

```diff
@@ -196,3 +197,58 @@ operator << (ostream &out, AudioSound::SoundStatus status) {
 
   return out << "**invalid AudioSound::SoundStatus(" << (int)status << ")**";
 }
+
+/**
+ * Get the comment attached to this AudioSound as a list of strings.
+ */
+vector_string AudioSound::
+get_raw_comment() const {
+  return vector_string();
+}
+
+/**
+ * Returns true if this AudioSound has a comment with the given key,
+ * i.e. "author". Case-sensitive.
+ */
+bool AudioSound::
+has_comment(std::string &key) const {
```

Comment:
This should be `const std::string &`, requiring a mutable reference is silly and probably wouldn't be wrapped by interrogate.